website/docs/components/notice-bubble/examples/special_events_notice.tsx

```diff
@@ -11,7 +11,7 @@ const Demo = () => {
       children: 'Your post is on the way - we will take care!',
```

Comment:
I would propose changing this text to something longer, such as: "Your post is on its way, and we will take great care of it!" — or something similar, to make the example more beautiful.

Reply:
done!